ext/native/thin3d/VulkanRenderManager.cpp

```diff
@@ -612,6 +613,70 @@ bool VulkanRenderManager::InitDepthStencilBuffer(VkCommandBuffer cmd) {
 	return true;
 }
 
+static void RemoveDrawCommands(std::vector<VkRenderData> *cmds) {
+	// Here we remove any DRAW type commands when we hit a CLEAR.
+	cmds->erase(std::remove_if(cmds->begin(), cmds->end(), [](const VkRenderData &data) {
+		return data.cmd == VKRRenderCommand::DRAW || data.cmd == VKRRenderCommand::DRAW_INDEXED;
+	}), cmds->end());
+}
+
+static void CleanupRenderCommands(std::vector<VkRenderData> *cmds) {
+	size_t lastCommand[256];
```

Comment:
Could just define a max value in the enum and use that here, 256 is too many for sure.

Reply:
I'm not sure it'll break the bank - wanted to avoid all the annoying default switch values for such enums.  Personally I think it's silly that c++11 still doesn't just have a max value automatically, especially for class enums.

-[Unknown]

Reply:
We do loop over this array at some point, it will eat a tiny bit of cpu :) but yeah memory-wise it shouldn't be a big deal... Meh.